packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -237,7 +147,59 @@ export class Tabs {
     // pass all our new aria information to each `<calcite-tab>` and
     // `<calcite-tab-title>` which will check if they can update their internal
     // `controlled` or `labeledBy` states and re-render if necessary
-    this.tabs.forEach((el) => el.updateAriaInfo(tabIds, titleIds));
+    tabs.forEach((el) => el.updateAriaInfo(tabIds, titleIds));
     this.titles.forEach((el) => el.updateAriaInfo(tabIds, titleIds));
   }
+
+  private updateItems(): void {
+    const { position, scale } = this;
+
+    const nav = this.el.querySelector("calcite-tab-nav");
+    if (nav) {
+      nav.position = position;
+      nav.scale = scale;
+    }
+
+    Array.from(this.el.querySelectorAll("calcite-tab")).forEach((tab: HTMLCalciteTabElement) => {
+      if (tab.parentElement === this.el) {
+        tab.scale = scale;
+      }
+    });
+
+    Array.from(this.el.querySelectorAll("calcite-tab-nav > calcite-tab-title")).forEach(
```

Comment:
nitpick: since `nav` is already queries for above. could you do `Array.from(nav.querySelectorAll("calcite-tab-title")).forEach(`

Reply:
This code is carried over from `dev`, just relocated in the file.